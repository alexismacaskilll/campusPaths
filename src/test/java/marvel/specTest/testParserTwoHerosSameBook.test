# Test the parser on a file that contains two heros in the same book

# Load a graph from tsv file that contains two heros in the same book
LoadGraph twoHerosSameBook twoHerosSameBook.tsv

# Print the heros in the graph
ListNodes twoHerosSameBook

# Print the outgoing edges from hero1
ListChildren twoHerosSameBook hero1
# Print the outgoing edges from hero2
ListChildren twoHerosSameBook hero2