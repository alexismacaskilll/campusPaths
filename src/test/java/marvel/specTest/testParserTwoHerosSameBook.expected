# Test the parser on a file that contains two heros in the same book

# Load a graph from tsv file that contains two heros in the same book
loaded graph twoHerosSameBook

# Print the heros in the graph
twoHerosSameBook contains: hero1 hero2

# Print the outgoing edges from hero1
the children of hero1 in twoHerosSameBook are: hero2(book1)
# Print the outgoing edges from hero2
the children of hero2 in twoHerosSameBook are: hero1(book1)